templates/App/Datum/_datum.html.twig

```diff
@@ -56,27 +77,7 @@
             </div>
         {% endif %}
 
-        <div class="input-field outlined col s10 offset-s1 m5">
-            <select id="data_{{ iteration }}_visibility"
-                    name="{{ entity }}[data][{{ iteration }}][visibility]"
-                    data-controller="select--field-visibility"
-            >
-            {% for visibility in getVisibilities() %}
-                <option value={{ visibility }}
-                        {% if datum is defined and datum.visibility == visibility %}
-                            selected
-                        {% elseif template is defined %}
-                            {% for field in template.fields %}
-                                {% if field.name == label and field.visibility == visibility %}
-                                    selected
-                                {% endif %}
-                            {% endfor %}
-                        {% endif %}>
-                    {{ ('global.visibilities.' ~ visibility)|trans }}
-                </option>
-            {% endfor %}
-            </select>
-            <label for="data_{{ iteration }}_visibility">{{ 'label.visibility'|trans }}</label>
-        </div>
+        {{ include('App/Datum/_' ~ type ~ '.html.twig') }}
     </div>
 </div>
+<br/>
```

Comment:
```css
diff --git a/assets/styles/main.css b/assets/styles/main.css
index 3f05544a..c30cb19b 100644
--- a/assets/styles/main.css
+++ b/assets/styles/main.css
@@ -1690,7 +1690,7 @@ input[type=color] {
 }
 
 form .row {
-    margin-bottom: 0;
+    margin-bottom: 15px;
 }
 ```
 
 or
 
 ```html
 diff --git a/templates/App/Datum/_datum.html.twig b/templates/App/Datum/_datum.html.twig
index 126f85d8..4a37e408 100644
--- a/templates/App/Datum/_datum.html.twig
+++ b/templates/App/Datum/_datum.html.twig
@@ -1,6 +1,6 @@
 {% set hasChoiceList = choiceList is defined and choiceList is not null %}
 
-<div {% if template is defined %} data-template="{{ template.name }}"{% endif %} class="datum row" data-additional-data-ta
rget="datum" data-iteration="{{ iteration }}">
+<div {% if template is defined %} data-template="{{ template.name }}"{% endif %} class="datum row" data-additional-data-ta
rget="datum" data-iteration="{{ iteration }} style="margin-bottom: 15px;">
     <div class="col s12 l10 offset-l1 xl8 offset-xl2 row">
         <input type="hidden"
 ```
 
Should work fine. If don't want to apply elsewhere, please create additional css class.
 
 Please consider